resources/lang/en.json

```diff
@@ -33,8 +33,7 @@
     "This version of the release is planned only and <strong>not released yet!</strong>": "This version of the release is planned only and <strong>not released yet!</strong>",
     "The estimated release date is :date": "The estimated release date is :date",
     "To learn more about Laravel's versioning strategy, check out the :link.": "To learn more about Laravel's versioning strategy, check out the :link.",
-    "Update <em>at least</em> to a security-maintained version <strong>as soon as possible!</strong>": "Update <em>at least</em> to a security-maintained version <strong>as soon as possible!</strong>",
-    "Update <em>at least</em> to a security-maintained version <strong>as soon as possible!</strong><br>\n                        The lowest version still getting security fixes is: :link <br>\n                        To upgrade, follow the instructions in the docs or use :link-laravelshifts to upgrade automatically.": "Update <em>at least</em> to a security-maintained version <strong>as soon as possible!</strong><br>\nThe lowest version still getting security fixes is: :link <br>\nTo upgrade, follow the instructions in the docs or use :link-laravelshifts to upgrade automatically.",
+    "Update <em>at least</em> to a security-maintained version <strong>as soon as possible!</strong><br>\n            The lowest version still getting security fixes is: :link <br>\n            To upgrade, follow the instructions in the docs or use :link-laravelshifts to upgrade automatically.": "Update <em>at least</em> to a security-maintained version <strong>as soon as possible!</strong><br>\nThe lowest version still getting security fixes is: :link <br>\nTo upgrade, follow the instructions in the docs or use :link-laravelshifts to upgrade automatically.",
```

Comment:
I separated this out to two strings on purpose; I don't like having the line break inline. But it looks like I *started* it and never finished it? Oops! It's OK if we keep it as three lines in one entry, but we have to be careful about how it handles the indentation after the newlines, which is why I was originally trying to separate it out.